_sass/hero.scss

```diff
@@ -1,12 +1,13 @@
 
 .hero {
   width: 100%;
-  background: linear-gradient(180deg, #238DFF, #2870F4);
+  background: $color-blue;
```

Comment:
Why switch to plain raw color, gradient is not fine ? (I prefer the the gradient, but this is only my personal opinion)

Reply:
I made this to stay consistent with the palette that was defined for template using data.gouv.fr color scheme. The gradient was actually used before template.data.gouv.fr was created.

We can leave the gradient for now but it might get a bit too bright once the color scheme get adopted on the other platforms.

Reply:
:+1: 
OK, let's find another solution later.